<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tta_data</class>
 <widget class="QWidget" name="tta_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="tta_gridGroupBox">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="analyze_pushButton">
        <property name="toolTip">
         <string>Perform the fourier transform.</string>
        </property>
        <property name="text">
         <string>Analyze data</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Frequency domain</string>
      </attribute>
      <widget class="QGroupBox" name="plot1_GroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>1314</width>
         <height>287</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="horizontalGroupBox">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="freq_peaks_label">
             <property name="text">
              <string>Signal peaks:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="current_peak_comboBox">
             <property name="toolTip">
              <string>Select the peak at the displayed index.</string>
             </property>
             <property name="placeholderText">
              <string>Select the peak at the displayed index.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="peak_range_label">
             <property name="text">
              <string>Points around peak:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="peak_range_spinBox">
             <property name="toolTip">
              <string>Width around the peak that should be displayed.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="peak_threshold_label">
             <property name="text">
              <string>Peak threshold:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="peak_threshold_spinBox">
             <property name="toolTip">
              <string>Minimum height of the peak as multiplicative factor of the data mean.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="peak_separation_label">
             <property name="text">
              <string>Peak separation:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="peak_separation_spinBox">
             <property name="toolTip">
              <string>Minimum number of data points between neighboring peaks.</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="plot1_splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="PlotWidget" name="plot1_PlotWidget">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
          </widget>
          <widget class="FitWidget" name="plot1_fitwidget" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Time domain</string>
      </attribute>
      <widget class="QGroupBox" name="plot2_GroupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>1314</width>
         <height>308</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="plot2_splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="PlotWidget" name="plot2_PlotWidget"/>
          <widget class="FitWidget" name="plot2_fitwidget" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>FitWidget</class>
   <extends>QWidget</extends>
   <header>qudi.util.widgets.fitting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>plot1_PlotWidget</tabstop>
  <tabstop>plot2_PlotWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
